app/assets/stylesheets/data_sets.css.scss

```diff
@@ -2,3 +2,48 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
```

Comment:
Don't redeclare border styles. Use sassy's $ syntax to define local variables, including on for the border style. An example, is in visualizations.css.scss